WhenPushComesToShove/Assets/3-Scripts/MinigameLogic/SumoLogic.cs

```diff
@@ -20,6 +20,15 @@ void Start()
         
     }
 
+    public override void Init()
+    {
+        base.Init();
+
+        foreach (HealthBar b in GameState.playerHealthBars)
```

Comment:
Do we not already have a more general solution for hiding healthbars? It might be tied to disabling damage in which case it would make sense to do it this way. Are we sure that this cleans up properly and turns back on for the next minigames?

Reply:
Disabling damage made it so the players couldn't get harmed by the spikes. Yep, and this does clean up. The healthbars are reenabled in games where they are needed